packages/cli/src/commands/init.ts

```diff
@@ -33,8 +34,10 @@ async function action(projectName: string, version: string, options: any): Promi
 
   const dependencies = link ? link.split(',') : [];
   const flags = { dependencies, installDependencies, force, publish };
+  const initArguments = { ...prompted, ...flags };
 
-  await init({ ...prompted, ...flags });
+  await init(initArguments);
+  await Telemetry.report('init', initArguments, interactive);
```

Comment:
In all other commands, the telemetry report happens before the actual command. Is there a reason to swap them here?

Reply:
I think we swapped them because before a project is initialized there is no project file to read or store the project-local preference. But now that I think of it that doesn't make a lot of sense, because you can also run `compile` without running `init`. So we should handle the scenario where there is no project file for local preference.